### PR TITLE
Prune CA interface.

### PR DESCRIPTION
**Pull request by myidpt**

### CODE REVIEW

File: security/pkg/pki/ca/ca.go [hunk 1]
```diff
@@ -176,21 +169,7 @@ func NewIstioCA(opts *IstioCAOptions) (*IstioCA, error) {
 
 // Sign takes a PEM-encoded CSR and returns a signed certificate. If the CA is a multicluster CA,
```
Comment (incfly) [inline security/pkg/pki/ca/ca.go]: Could you update the comments as well? This now can only talks about forCA is true or not.

Comment (myidpt) [reply security/pkg/pki/ca/ca.go]: Done.

**Pull request merged**